test/cucumber/steps/CompararNotasSteps.groovy

```diff
@@ -0,0 +1,119 @@
+package steps
+
+import pages.*
+import rpa.GrupoPesquisadores
+import rpa.GrupoPesquisadoresController
+import rpa.Pesquisador
+import rpa.PesquisadorController
+
+import static cucumber.api.groovy.EN.*
+this.metaClass.mixin(cucumber.api.groovy.Hooks)
+this.metaClass.mixin(cucumber.api.groovy.EN)
+//Cenario 1
+Given(~'^Os pesquisadores "([^"]+)" de cpf "([^"]+)" e "([^"]+)" de cpf "([^"]+)" estão cadastrados no sistema$'){String nome1,String cpf1,String nome2,String cpf2->
+    to CreateResearcherPage
+    at CreateResearcherPage
+    page.dataToAdd(nome1,cpf1)
+    assert Pesquisador.findByCpf(cpf1)!=null
+    to CreateResearcherPage
+    at CreateResearcherPage
+    page.dataToAdd(nome2,cpf2)
+    assert Pesquisador.findByCpf(cpf2)!=null
+}
```

Comment:
codigo grande, acredito que dava pra ser simplificado, fazendo os dois dataToAdd em sequencia e usando apenas um assert com um &&(and)